net-im/slack++/slack++-0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Enterprise chat and services on the command line - slack++"
+DESCRIPTION="slack++ is an ncurses based client the Searchable Log of All Conversation and Knowledge"
+HOMEPAGE="https://github.com/ga2arch/slack"
+COPYRIGHT="2018"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="961dec741764dfb81999703b84989d93ec7a88ee"
+SOURCE_URI="https://github.com/ga2arch/slack/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9c84b5d0ee7b528edfdde219263dd7c307c6e8cd9aadc491e7148f7457ea30a5"
+SOURCE_DIR="slack-$srcGitRev"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+PROVIDES="slack++$secondaryArchSuffix = $portVersion
+		cmd:slack++$secondaryArchSuffix = $portVersion"
+
+REQUIRES="
+ 	haiku$secondaryArchSuffix
+	lib:libjsoncpp$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libjsoncpp$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+"
```

Comment:
```diff
-"
+	"

Reply:
This needs to be done ....

Reply:
Sorry brain did not parse what you were showing me.

Reply:
The double quote needs a tab in front of it :)

Reply:
Yeah I also fixed the provides which was slightly janky.